lib/node_modules/@stdlib/fft/base/fftpack/lib/passb3.js

```diff
@@ -0,0 +1,152 @@
+*       notices, this list of conditions, and the disclaimer below.
+*
+*     - Redistributions in binary form must reproduce the above copyright
+*       notice, this list of conditions, and the disclaimer below in the
+*       documentation and/or other materials provided with the
+*       distribution.
+*
+* THIS SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+* EXPRESS OR IMPLIED, INCLUDING, BUT NOT LIMITED TO THE WARRANTIES OF
+* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+* NONINFRINGEMENT. IN NO EVENT SHALL THE CONTRIBUTORS OR COPYRIGHT
+* HOLDERS BE LIABLE FOR ANY CLAIM, INDIRECT, INCIDENTAL, SPECIAL,
+* EXEMPLARY, OR CONSEQUENTIAL DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+* ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+* CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS WITH THE
+* SOFTWARE.
+* ```
+*/
+
+/* eslint-disable max-len */
+
+'use strict';
+
+// MODULES //
+
+var chRef = require( './ch_ref.js' );
+var ccRef = require( './cc_ref.js' );
+
+
+// VARIABLES //
+
+var taur = 0.5;
+var taui = 0.866025403784439;
+
+
+// MAIN //
+
+/**
+* Performs a pass of length 3 of the FFT algorithm.
+*
+* @private
+* @param {number} ido - number of real values for each transform
+* @param {number} l1 - length of the input sequences
+* @param {Float64Array} cc - input array containing sequences to be transformed
+* @param {Float64Array} ch - output array containing transformed sequences
+* @param {Float64Array} wa1 - first array of twiddle factors
+* @param {Float64Array} wa2 - second array of twiddle factors
+* @returns {void}
+*/
+function passb3( ido, l1, cc, ch, wa1, wa2 ) {
+	var chOffset;
+	var ccOffset;
+	var ci2;
+	var ci3;
+	var di2;
+	var di3;
+	var cr2;
+	var cr3;
+	var dr2;
+	var dr3;
+	var ti2;
+	var tr2;
+	var i;
+	var k;
+
+	// Parameter adjustments...
+	chOffset = 1 + ( ido * ( 1 + l1 ) );
+	ccOffset = 1 + ( ido << 2 );
+
+	// Function body:
+	if ( ido === 2 ) {
+		for ( k = 1; k <= l1; ++k ) {
+			tr2 = cc[ ccRef( 1, 2, k, 3, ido ) - ccOffset ] + cc[ ccRef( 1, 3, k, 3, ido ) - ccOffset ];
+			cr2 = cc[ ccRef( 1, 1, k, 3, ido ) - ccOffset ] + ( taur * tr2 );
+			ch[ chRef( 1, k, 1, l1, ido ) - chOffset ] = cc[ ccRef( 1, 1, k, 3, ido ) - ccOffset ] + tr2;
+			ti2 = cc[ ccRef( 2, 2, k, 3, ido ) - ccOffset ] + cc[ ccRef( 2, 3, k, 3, ido ) - ccOffset ];
+			ci2 = cc[ ccRef( 2, 1, k, 3, ido ) - ccOffset ] + ( taur * ti2 );
+			ch[ chRef( 2, k, 1, l1, ido ) - chOffset ] = cc[ ccRef( 2, 1, k, 3, ido ) - ccOffset ] + ti2;
+			cr3 = taui * ( cc[ ccRef( 1, 2, k, 3, ido ) - ccOffset ] - cc[ ccRef( 1, 3, k, 3, ido ) - ccOffset ] );
+			ci3 = taui * ( cc[ ccRef( 2, 2, k, 3, ido ) - ccOffset ] - cc[ ccRef( 2, 3, k, 3, ido ) - ccOffset ] );
+			ch[ chRef( 1, k, 2, l1, ido ) - chOffset ] = cr2 - ci3;
+			ch[ chRef( 1, k, 3, l1, ido ) - chOffset ] = cr2 + ci3;
+			ch[ chRef( 2, k, 2, l1, ido ) - chOffset ] = ci2 + ci3;
+			ch[ chRef( 2, k, 3, l1, ido ) - chOffset ] = ci2 - cr3;
+		}
```

Comment:
You can use an early `return` here so you can remove the `else` and reduce indentation.